pkg/operator/configobservation/configobservercontroller/observe_config_controller.go

```diff
@@ -138,6 +143,14 @@ func NewConfigObserver(
 			network.ObserveServicesSubnet,
 			network.ObserveExternalIPPolicy,
 			network.ObserveServicesNodePortRange,
+			nodeobserver.NewLatencyProfileObserver(
+				node.LatencyConfigs,
+				nodeobserver.NewSuppressConfigUpdateUntilSameProfileFunc(
+					operatorClient.(v1helpers.StaticPodOperatorClient),
```

Comment:
update `NewConfigObserver` to accept `operatorClient v1helpers.StaticPodOperatorClient`. I think it is a simple change.

Reply:
Changes done!